src/test/java/org/folio/rest/api/TestBase.java

```diff
@@ -78,4 +79,40 @@ void assertGetNotFound(URL url) {
       throw new RuntimeException(e);
     }
   }
+
+  void executeSqlFile(String fileContent)
```

Comment:
What is the difference between SQL that comes from a file and elsewhere? At this point they are both represented as strings?

Reply:
Yes, I use different RMB function to execute file, it splits the file content to lines, and executes it line by line.

Reply:
Is this really about it coming from a file, or that it includes multiple SQL statements?

Reply:
It coming from a file, for example migration scripts (alterHridSequences.sql file or hridSettings.sql)

Reply:
I understand that it is coming from a file. I'm trying to understand if the special processing is because it's come from a file or because it is multiple statements?

Neither of these methods can know that it has come from a file as they take String parameters.

Reply:
I think because it is multiple statements.

Reply:
Ok, then I think we should reflect that in the method name (it's a shame the RAML Module Builder method name cannot be so easily changed).

It's not major, so shouldn't block the merging of this change.

Reply:
Does `executeMultipleSqlStatements` work well?

Reply:
I'm good with that